pquery-run.sh

```diff
@@ -1352,14 +1372,25 @@ pquery_test(){
         SQL_FILE="--infile=${INFILE}"
       else
         # Multi-threaded run using a chunk from INFILE (${THREADS} clients)
-        echoit "Taking ${MULTI_THREADED_TESTC_LINES} lines randomly from ${INFILE} as testcase for this multi-threaded trial..."
+        if [ ${PQUERY3} -eq 1 ]; then
+          if [ "${TRIAL}" == "1" ]; then
+            echoit "Creating metadata randomly using random seed..."
+          else
+            echoit "Loading metadata from ${WORKDIR}/step_$((${TRIAL}-1)).dll ..."
+          fi
+        else
+          echoit "Taking ${MULTI_THREADED_TESTC_LINES} lines randomly from ${INFILE} as testcase for this multi-threaded trial..."
+        fi
         shuf --random-source=/dev/urandom ${INFILE} | head -n${MULTI_THREADED_TESTC_LINES} > ${RUNDIR}/${TRIAL}/${TRIAL}.sql
         SQL_FILE="--infile=${RUNDIR}/${TRIAL}/${TRIAL}.sql"
       fi
-      # Debug echo "-------"; cat ${RUNDIR}/${TRIAL}/${TRIAL}.sql; echo "-------"
```

Comment:
since we are not testing/fixing PXC as part of this JIRA. 
Better to ${PQUERY3} -eq 0 ] inside if [[ ${PXC} -eq 0 && ${GRP_RPL} -eq 0 ]